package.json

```diff
@@ -26,7 +26,7 @@
     "build": "npm run build:only",
     "build:only": "npm run tsc && cp package.json build && browserify src/index.ts -p tsify -d -s vl | exorcist build/vega-lite.js.map > build/vega-lite.js",
     "postbuild": "node node_modules/uglify-js/bin/uglifyjs build/vega-lite.js -cm --source-map content=build/vega-lite.js.map,filename=build/vega-lite.min.js.map -o build/vega-lite.min.js && npm run schema",
-    "build:examples": "npm run build:only && npm run build:examples-only",
+    "build:examples": "npm run data && npm run build:only && npm run build:examples-only",
```

Comment:
Note: This seems to make build:examples slower, which is a bit annoying. I'll merge this for now, but I wonder if we should include `node_modules/vega-datasets` on jekyll instead of copying data.